features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -739,6 +739,7 @@
                 Supported versions: IS 5.4.0 onwards
             -->
             <RequestObjectValidator>{{oauth.oidc.extensions.request_object_validator}}</RequestObjectValidator>
+            <CIBARequestObjectValidator>{{oauth.oidc.extensions.ciba_request_object_validator}}</CIBARequestObjectValidator>
```

Comment:
wrap with is defined block

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/3829/commits/de1080ae0a7afaee3d54afaa695a00d04116f76b